contracts/pools/FactoryRouter.sol

```diff
@@ -582,26 +583,6 @@ contract FactoryRouter is BFactory {
 
     // one single call to buy multiple DT for multiple assets (better UX, better gas optimization)
 
-    enum operationType {
```

Comment:
This looks like another example of a PR - a refactoring, probably a useful one, but not towards the goal of this ticket. As mentioned above, this makes it hard to review what this ticket is doing. This change should be in a ticket & PR specifically towards its goals.

Reply:
OK. Thanks.

This also would be something normally out of scope of this issue. Something to avoid in the future. (And perhaps here if the main reviewer decides.)

Reply:
I see. Makes sense.